lib/code-caller/code-caller-container.js

```diff
@@ -418,15 +418,15 @@ class CodeCallerContainer {
 
   _handleStdout(data) {
     this.debug('enter _handleStdout()');
-    this.outputStdout += data;
-    if (this.outputStdout.indexOf('\n') >= 0) {
+    this.outputStdout.push(data);
+    if (data.indexOf('\n') >= 0) {
       this._callIsFinished();
     }
     this.debug('exit _handleStdout()');
   }
   _handleStderr(data) {
     this.debug('enter _handleStderr()');
-    this.outputStderr += data;
+    this.outputStderr.push(data);
```

Comment:
Where do we do `this.outputStderr.join('')`? I can't see it anywhere.

Reply:
We actually never use it at all (I had to double-check). I guess we could remove it entirely, though I like having it around for potential debugging (I can just add `console.log(this.outputStderr.join(''))` to finish handler).

Reply:
Ok, sounds good, I was just worried we were missing it.